### PR TITLE
First Pull Request

### PR DESCRIPTION
**Pull request by Zven94**
### **Conference Page**

change add in this branch

- Added html file
- Added two .css styles for mobile and desktop screen
- Added js for events
- Added linters
- Added README file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+
+    <section class="about-page" id="about-page">
+
+      <!--    titles    -->
+
+      <div class="about-page-tittles">
+        <h2 class="about-page-h2">"Security is the future"</h2>
+        <h1 class="about-page-h1">Block-chain first TECH SUMMIT 2023</h1>
+      </div>
+
```
Comment (Shedrack-Sunday) [inline index.html]: - [x] Based on project requirements, you're meant to build two pages. `The home page` and `The about page`. Kindly separate this section as a separate file.

## Requirements:
image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+      </div>
+    </section>
+    <section class="sponsor-page">
+      <div class="sponsor-page-past-sponsors">
+        <div><h1>Our Partners</h1></div>
+        <div class="sponsor-page-past-div"></div>
+        <div class="sponsor-img-1"><img src="image/airbnb 1.png" alt=""><img src="image/mozilla 1.png" alt=""><img src="image/naver 1.png" alt=""></div>
+        <div class="sponsor-img-2"><img src="image/googlekorea 1.png" alt=""><img src="image/daumkakaologo 1.png" alt=""></div>
+      </div>
+    </section>
```
Comment (Shedrack-Sunday) [inline index.html]: - [x] The `footer` sections of your project are not displaying. There are guidelines for how its meant to be designed based on the design given. when you separate the `about` page from the home, kindly follow this designs and make sure its visible 👍 

## Home page Desktop:
image

## About page Desktop:
image

## Home/About page mobile:
image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+          <h2 class="home-page-h2">"Security is the future"</h2>
+          <h1 class="home-page-h1">Block-chain first TECH SUMMIT 2023</h1>
+        </div>
+
+        <!--    paragraph    -->
+
+        <div class="home-page-text">
+          <p class="home-page-text-p">Join us into the biggest congrest about block-chain technology
+            and know the new developments and improvements with companies
+            that are already developing applications that will renew the world
```
Comment (Shedrack-Sunday) [inline index.html]: - [x] Good job here 👏 . But in comparison to the original design, kindly align the text to the left with lo as you can see on the design given. This makes its have a better visual appeal and a good user experience 👍 

## Your work:
image

## Design given:
image

File: index.html [hunk 4]
```diff
@@ -0,0 +1,227 @@
...
+          <div class="burger-icon line1"></div>
+          <div class="burger-icon line2"></div>
+          <div class="burger-icon line3"></div>
+        </button>
+        <!--burger menu-->
+        <ul class="burger-menu">
+          <div class="burger-menu-style">
+              <li class="burger-menu-exit-button"><button class="exitButton"></button></li>
+              <li class="burger-menu-homePage-button"><a href="#home-page" class="burger-menu-home-page">Home Page</a></li>
+              <li class="burger-menu-aboutPage-button"><a href="#about-page" class="burger-menu-aboutPage-button">About</a></li>
```
Comment (Shedrack-Sunday) [inline index.html]: - [x] For both `Desktop, mobile` in the home and about, kindly refactor the header section using the `nav` tag to insert the social links, logo, and other navigation links. 
- This is an industry best practice which has to be followed to make you a better developer 👍

File: index.html [hunk 5]
```diff
@@ -0,0 +1,227 @@
...
+          <li><a href="" class="desktop-menu-about">Join</a></li>
+          <li><a href="" class="desktop-menu-about">Sponsors</a></li>
+          <li><a href="" class="desktop-menu-about">News</a></li> 
+        </ul>        
+      </section>
+      <div class="nav-logo-container">
+        <a href="#home-page" class="nav-logo-link">
+          <div class="nav-logo-img"></div>
+            <h1 class="nav-logo">Libra</h1>
+        </a>
```
Comment (Shedrack-Sunday) [inline index.html]: - [x] Only one `h1` tag should be available in your codebase. 
- Using only one `h1` tag in a project limits accessibility, SEO optimization, and content organization. Multiple heading tags (such as `h2`, `h3`, etc.) should be used to create a hierarchical structure, improve search engine rankings, and enhance the user experience. The appropriate use of heading tags facilitates better comprehension, navigation, and clarity of the project's content.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,227 @@
...
+          <h2 class="home-page-info-date">2023.10.15 ~ 16</h2>
+          <h3 class="home-page-info-place">see you on: Dubai Exhibition Centre</h3>
+        </div>
+    </section>
+
+      <!--    schedule main program    -->
+
+      <div class="home-page-main-program">        
+        <div class="home-page-main-program-title-content">
+          <h1 class="home-page-main-program-title">Main Program</h1>
```
Comment (Shedrack-Sunday) [inline index.html]: - [x] Kindly refer to my comments on line `59` on the use of only one `h1` tag for you project 👍

File: index.html [hunk 7]
```diff
@@ -0,0 +1,227 @@
...
+        <button class="home-page-info-joinNowBtn">GET YOUR PASS</button>
+
+      </div>
+    </section>
+      
+      <!--    speaker me page   -->
+    
+    <section class="speaker-page">
+      <h1 class="speaker-page-h1">Featured Speaker</h1>
+      <div class="speaker-page-h1-div"></div>
```
Comment (Shedrack-Sunday) [inline index.html]: - [x] Kindly refer to my comments on line `59` on the use of only one `h1` tag for you project 👍

File: index.html [hunk 8]
```diff
@@ -0,0 +1,227 @@
...
+            the logo competition from 8, June to 7, July.</h2>
+      </div>
+      <!--  my logo-->  
+      <div class="about-page-past-logo">
+        <img src="image/logo-no-circle.png" alt="" width="100px">
+        <h1 class="about-page-past-logo-text">Libra</h1>
+      </div>
+      <!--    --> 
+      <div class="about-page-past">
+        <h1 class="about-page-past-h1">See the past Block-chain TECH SUMMIT 2023</h1>
```
Comment (Shedrack-Sunday) [inline index.html]: - [x] Kindly refer to my comments on line `59` on the use of only one `h1` tag for you project 👍

File: index.html [hunk 9]
```diff
@@ -0,0 +1,227 @@
...
+        <!-- agregar background img -->
+        <div class="about-page-past-background-2">
+         <h1 class="about-page-past-media-h1-1">2020</h1>
+         <h2 class="about-page-past-media-h2-2">Second Congres in Germany</h2>
+        </div>
+      </div>
+    </section>
+    <section class="sponsor-page">
+      <div class="sponsor-page-past-sponsors">
+        <div><h1>Our Partners</h1></div>
```
Comment (Shedrack-Sunday) [inline index.html]: - [x] Kindly refer to my comments on line `59` on the use of only one `h1` tag for you project 👍

Comment (Shedrack-Sunday) [inline index.html]: - [x] Kindly refer to my comments on line `59` on the use of only one `h1` tag for you project 👍

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hello @Zven94       👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well so far in the project,  However, there are a few you need to work on, to move to the next project. 💯

 


## Highlights   
- Good use of GitHub features ✔️   
- Nice use of appropriate colors 👍 

  
# Required Changes  🌵 

- [x] You made some impressive implementations which I must say is a job well done. However, there are a few features you missed or need to adjust.
- One of which is to have separate files for the `Home` and `About` pages
- Second, is to use one `h1` tage per page. You have multiple 


_Check the comments for a detailed review 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: main.js [hunk 10]
```diff
@@ -0,0 +1,195 @@
...
+
+  const speakerH22 = document.createElement('h2');
+  speakerDiv2.appendChild(speakerH22);
+  speakerH22.classList.add('speaker-page-container-h2-2');
+  speakerH22.textContent = myArr[i].workDescription;
+}
+
+for (let i = 0; i <= 5; i += 1) {
+  creatSpeakerSection(i);
+}
```
Comment (divinecharlotte) [inline main.js]: - [x] kindly consider sticking to their provided design as much as possible to meet the below requirements
Screen Shot 2023-05-25 at 07 15 17
 check the images below and see the difference and consider fixing your design before you proceed
- your design 👇 
Screen Shot 2023-05-25 at 07 17 59


- check everything I pointed below in the  given design and fix it👇 
Screen Shot 2023-05-25 at 07 19 16

File: home.html [hunk 11]
```diff
@@ -0,0 +1,171 @@
...
+      </div>
+    </section>
+    <section class="sponsor-page">
+      <div class="sponsor-page-past-sponsors">
+        <div><h2>Our Partners</h2></div>
+        <div class="sponsor-page-past-div"></div>
+        <div class="sponsor-img-1"><img src="image/airbnb 1.png" alt=""><img src="image/mozilla 1.png" alt=""><img src="image/naver 1.png" alt=""></div>
+        <div class="sponsor-img-2"><img src="image/googlekorea 1.png" alt=""><img src="image/daumkakaologo 1.png" alt=""></div>
+      </div>
+    </section>
```
Comment (divinecharlotte) [inline home.html]: - [x] kindly consider making sure that this section looks exactly like the one provided in the design to make your work an exact duplicate of the given design
- your design 👇 
Screen Shot 2023-05-25 at 07 29 01


1. consider displaying all 5 partners on the same line as in the given design 
2. consider adding the red line under the our partners title

File: home.html [hunk 12]
```diff
@@ -0,0 +1,171 @@
...
+        </ul>        
+      </section>
+      <div class="nav-logo-container">
+        <a href="home.html" class="nav-logo-link">
+          <div class="nav-logo-img"></div>
+            <h2 class="nav-logo">Libra</h2>
+        </a>
+      </div>
+    </nav>
+  </header>
```
Comment (divinecharlotte) [inline home.html]: kindly consider checking on the image below to stick to the given design as much as possible
Screen Shot 2023-05-25 at 07 31 31

File: home.html [hunk 13]
```diff
@@ -0,0 +1,171 @@
...
+          <div class="home-page-main-program-5">
+            <img src="image/program_icon_05.png" alt="" class="home-page-main-program-img">
+            <h2 class="home-page-main-program-5-oratory">Ignite</h2>
+            <p class="home-page-main-program-paragraph"> get opportunities to network with CC affiliates around the world, also after the summit.</p>
+          </div>
+        </div>
+
+          <!--    button to submit    -->
+
+        <button class="home-page-info-joinNowBtn">GET YOUR PASS</button>
```
Comment (divinecharlotte) [inline home.html]: kindly consider removing the borders on this button and just underline it to me the design requirements

**Review by divinecharlotte — CHANGES_REQUESTED** [review]
Hi @Zven94       , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## To Highlight 💯 

- linters are passing 👍 
- Descriptive Pr 👍 




 

## Required Changes ♻️
_Check the comments under the review._
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 14]
```diff
@@ -0,0 +1,178 @@
...
+        
+        <button class="bugerBtn">
+          <div class="burger-icon line1"></div>
+          <div class="burger-icon line2"></div>
+          <div class="burger-icon line3"></div>
+        </button>
+        <!--burger menu-->
+        <ul class="burger-menu">
+          <div class="burger-menu-style">
+              <li class="burger-menu-exit-button"><button class="exitButton"></button></li>
```
Comment (tufoinnkuo10) [inline index.html:39]: - [optional] wonderful job implementing the changes requested by the previous reviewer. as a bonus to make your work better, kindly add a margin left to the main headline section and align with your logo so it matches the original design.
- Also, please add the red line under the main program heading and add little margin right to the desktop nav items like it the design. nice work
yours
Screenshot (1088)_LI
original
Screenshot (1090)

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:✔️✔️ 

Hi @Zven94   ✔️✔️ ✔️✔️ 

Your project is complete! Great job for taking time to make the changes🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request✔️✔️ 
- Behance design well followed ✔️✔️ 
- well deployed✔️✔️ 
- work well presented✔️✔️ 
- No linter errors✔️✔️ 


---[optional] please always ensure to have a good PR title/description so it can portray what you worked on from a glance.
<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

smile

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 14 above)
Comment (Zven94) [reply index.html:39]: Thank you for your time! I already addded the changes! 💪💪

**Pull request merged**